products/windowsServerCore.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Microsoft Windows Server Core
```

Comment:
suggestion to rename the files to windows-server-core, and windows-nano-server. 

We are trying to align filenames with permalink:  #5301

Reply:
I took Windows Server as base file. Adjustments should be done there then as-well.

Reply:
Yes, those are already present in #5301